packages/insomnia/src/common/analytics.ts

```diff
@@ -73,23 +73,41 @@ export enum SegmentEvent {
   vcsSyncStart = 'VCS Sync Started',
   vcsSyncComplete = 'VCS Sync Completed',
   vcsAction = 'VCS Action Executed',
+  buttonClick = 'Button Clicked',
 }
 
-type PushPull = 'push' | 'pull';
+/**
+ * You may refer to this Segment API Document - https://segment.com/docs/connections/spec/track/#properties
+ * to understand what 'event properties' is for.
+ *
+ * The following properties are custom defined attributes for our own metrics purpose to help
+ * ourselves to make product decisions
```

Comment:
```suggestion
 * The following properties are custom defined attributes for our own metrics purpose to help ourselves to make product decisions
```

no need to introduce arbitrary line wraps